qa/integration/services/logstash_service.rb

```diff
@@ -27,14 +34,15 @@ def initialize(settings)
       ls_version_file = YAML.load_file(LS_VERSION_FILE)
       ls_file = "logstash-" + ls_version_file["logstash"]
       # First try without the snapshot if it's there
-      @logstash_home = File.expand_path(File.join("../../../../build", ls_file), __FILE__)
+      @logstash_home = File.expand_path(File.join(LS_BUILD_DIR, ls_file), __FILE__)
       @logstash_home += "-SNAPSHOT" unless Dir.exists?(@logstash_home)
 
       puts "Using #{@logstash_home} as LS_HOME"
       @logstash_bin = File.join("#{@logstash_home}", LS_BIN)
       raise "Logstash binary not found in path #{@logstash_home}" unless File.file? @logstash_bin
     end
-
+    
+    @application_settings_file = File.join("#{@logstash_home}", LS_CONFIG_FILE)
```

Comment:
no need to cast it into a string before unless the `@logstash_home` can be nil? 
If its the case we can use `#to_s` instead of string interpolation.